test/e2e/specs/specs-playwright/wp-signup__free.ts

```diff
@@ -116,29 +114,11 @@ describe(
 		} );
 
 		describe( 'Delete user account', function () {
-			let newPage: Page;
-
-			it( 'Launch new context to ensure correct host', async function () {
-				newPage = await BrowserManager.newPage( { newContext: true } );
-			} );
-
-			it( 'Login', async function () {
-				// Logging in and immediately using navbar often leads to race conditions when the redirect
-				// from <rooturl> to <rooturl>/home/<site> redirect takes you out of the navigation you just did.
-				// We need to wait for that to happen before proceeding.
-				const expectedLandngUrl = DataHelper.getCalypsoURL( `home/${ selectedDomain }` );
-				const loginFlow = new LoginFlow( newPage, {
-					username: username,
-					password: signupPassword,
-				} );
-				await Promise.all( [
-					newPage.waitForNavigation( { url: expectedLandngUrl } ),
-					loginFlow.logIn(),
-				] );
-			} );
-
 			it( 'Close account', async function () {
-				const closeAccountFlow = new CloseAccountFlow( newPage );
+				// Magic link only works on production webapp.
+				// Revisit the webapp under test.
+				await page.goto( DataHelper.getCalypsoURL() );
+				const closeAccountFlow = new CloseAccountFlow( page );
```

Comment:
@dpasque I figured out that all that's required is to redirect the page to the value returned by `getCalypsoURL`. On Pre-Release tests this returns the `wpcalypso.wordpress.com` URL, and once navigated there we can proceed with the account deletion.

Removing the extraneous login step reduces the runtime by however many seconds the login process takes.

Here are the logs confirming that `wordpress.com` is redirected to `wpcalypso.wordpress.com` properly:

```
 navigated to "https://wordpress.com/log-in/link/use
 navigated to "https://wordpress.com/
...
navigating to "https://wpcalypso.wordpress.com/", waiting until "load"
 navigated to "https://wpcalypso.wordpress.com/home/e2eflowtesting1632438661097641.wordpress.com

```

Reply:
Also, I plan on moving this test out of `Signup: Free` suite and into its own standalone test, so this change will churn again soon.